CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `go.opentelemetry.io/otel/exporters/prometheus` exporter now correctly adds `_total` suffixes to counter metrics. (#3360)
 - The `go.opentelemetry.io/otel/exporters/prometheus` exporter now adds a unit suffix to metric names.
    This can be disabled using the `WithoutUnits()` option added to that package. (#3352)
+- The `go.opentelemetry.io/otel/exporters/prometheus` exporter fixes duplicated `_total` suffixes. (#3369)
```

Comment:
Please move this up into the `Unreleased` section.

Reply:
Done